integration/ledger/snapshot_test.go

```diff
@@ -0,0 +1,816 @@
+	Expect(bcInfo.BootstrappingSnapshotInfo.LastBlockInSnapshot).To(Equal(uint64(lastBlockInSnapshot)))
+
+	By("waiting for all peers to have the same ledger height")
+	channelPeers := n.PeersWithChannel(channelID)
+	maxLedgerHeight := nwo.GetMaxLedgerHeight(n, channelID, channelPeers...)
+	nwo.WaitUntilEqualLedgerHeight(n, channelID, maxLedgerHeight, channelPeers...)
+}
+
+func verifyQSCC(n *nwo.Network, peer *nwo.Peer, channelID string, lastBlockInSnapshot int, txidBeforeSnapshot string) {
+	peerID := peer.ID()
+	By("verifying qscc GetBlockByNumber returns an error for block number before snapshot on peer " + peerID)
+	resp := callQSCC(n, peer, "qscc", "GetBlockByNumber", 1, channelID, strconv.Itoa(lastBlockInSnapshot))
+	Expect(resp).To(ContainSubstring(fmt.Sprintf("The ledger is bootstrapped from a snapshot. First available block = [%d]", lastBlockInSnapshot+1)))
+
+	By("verifying qscc GetBlockByNumber succeeds for a block number after snapshot on peer " + peerID)
+	callQSCC(n, peer, "qscc", "GetBlockByNumber", 0, channelID, fmt.Sprintf("%d", lastBlockInSnapshot+1))
+
+	By("verifying qscc GetBlockByTxID returns an error for a txid before snapshot on peer " + peerID)
+	resp = callQSCC(n, peer, "qscc", "GetBlockByTxID", 1, channelID, txidBeforeSnapshot)
+	Expect(resp).To(ContainSubstring(fmt.Sprintf("Failed to get block for txID %s, error details for the TXID [%s] not available. Ledger bootstrapped from a snapshot. First available block = [%d]",
+		txidBeforeSnapshot, txidBeforeSnapshot, lastBlockInSnapshot+1)))
+
+	By("verifying qscc GetBlockByTxID succeeds for a txid after snapshot on peer " + peerID)
+	newTxid := getTxIDFromLastBlock(n, peer)
+	callQSCC(n, peer, "qscc", "GetBlockByTxID", 0, channelID, newTxid)
+
+	By("verifying qscc GetTransactionByID returns an error for a txid before snapshot on peer " + peerID)
+	resp = callQSCC(n, peer, "qscc", "GetTransactionByID", 1, channelID, txidBeforeSnapshot)
+	Expect(resp).To(ContainSubstring(fmt.Sprintf("Failed to get transaction with id %s, error details for the TXID [%s] not available. Ledger bootstrapped from a snapshot. First available block = [%d]",
+		txidBeforeSnapshot, txidBeforeSnapshot, lastBlockInSnapshot+1)))
+
+	By("verifying qscc GetTransactionByID succeeds for a txid after snapshot on peer " + peerID)
+	callQSCC(n, peer, "qscc", "GetTransactionByID", 0, channelID, newTxid)
+}
+
+func verifyChannelConfigUpdate(n *nwo.Network, orderer *nwo.Orderer, peer *nwo.Peer, channelID string) {
+	By("updating policy in channel config on peer " + peer.ID())
+	var org *nwo.Organization
+	for _, o := range n.Organizations {
+		if o.Name == peer.Organization {
+			org = o
+			break
+		}
+	}
+	currentConfig := nwo.GetConfig(n, peer, orderer, channelID)
+	updatedConfig := proto.Clone(currentConfig).(*common.Config)
+	updatedConfig.ChannelGroup.Groups["Application"].Groups[org.Name].Policies["Writers"].Policy.Value = protoutil.MarshalOrPanic(policydsl.SignedByMspMember(org.MSPID))
```

Comment:
At the integration level tests, I would avoid some dummy paths. If someone adds a legitimate check later, this will start failing. The channel config update will anyway gets tested when you upgrade the capability to 2_0 - I guess that we can avoid this function entirely.